RELEASING.md

```diff
@@ -189,18 +189,18 @@ steps:
 
 ### Release `5mins`
 
-- [ ] Ensure that all checks are green ✅ for the `<SDK_GIT_SHA>` on the `main`
+- [x] Ensure that all checks are green ✅ for the `<SDK_GIT_SHA>` on the `main`
   branch that you are about to release. This will usually be the commit that
   bumps the Engine version, the one that you merged earlier.
-- [ ] `3mins` When you have confirmed that all checks are green, run the following:
+- [x] `3mins` When you have confirmed that all checks are green, run the following:
 
 
 ```console
 # To find the previously released SDK version, go to:
 # https://github.com/dagger/dagger/releases?q=sdk%2Fgo&expanded=true
-# e.g. export GO_SDK_VERSION=v0.7.0
+# e.g. export GO_SDK_VERSION=v0.7.2
```

Comment:
Small question, why is it the only dependency with a different version?

Reply:
Go started at a different version that Node.js & Python. Both Node.js & Python became in sync with the Engine & CLI by accident. This provides more context:
- https://github.com/dagger/dagger/discussions/3456

FTR:
- Node.js SDK started from `v0.1.0` https://github.com/dagger/dagger/releases?expanded=true&page=3&q=sdk%5C%2Fnodejs
- Python SDK releases started from `v0.1.1`: https://github.com/dagger/dagger/releases?expanded=true&page=3&q=sdk%2Fpython%2Fv0 (`v0.1.0` was a bad release)
- Go SDK releases started from `v0.3.0` (since it was meant to be a continuation from Dagger Europa `v0.2.x`): https://pkg.go.dev/dagger.io/dagger?tab=versions